foundations/installations/setting_up_git.md

```diff
@@ -187,4 +187,5 @@ You probably felt like you were way in over your head, and you probably didn't u
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+*   [Understanding SSH Key Pairs](https://winscp.net/eng/docs/ssh_keys) SSH is an implimintation of public-key cryptography, also known as asymmetric cryptography. Having basic understanding of how it works can help you understand what SSH key is about.
+*   [Asymmetric Encryption - Simply expalined] a short video explaining Asymmetric Encription.
```

Comment:
If we're adding a video here, this should be a hyperlink.

Reply:
Yes sure, added the link. Thank you!

Reply:
It went into a second commit. hope this is the right way.